### PR TITLE
tflite ios create image segmentation exsample

### PR DESCRIPTION
**Pull request by st-tuanmai**

### CODE REVIEW

**Comment by st-tuanmai** [comment]
@PaulTR 
Hi Paul,
Now model selfie_segmenter has an error when creating tflite task.
`ImageSegmenter[692:59366] Encountered unresolved custom op: Convolution2DTransposeBias.
See instructions: https://www.tensorflow.org/lite/guide/ops_custom
ERROR: Encountered unresolved custom op: Convolution2DTransposeBias.
See instructions: https://www.tensorflow.org/lite/guide/ops_custom 
2024-04-22 11:26:58.522575+0700 ImageSegmenter[692:59366] Node number 244 (Convolution2DTransposeBias) failed to prepare.`
Then I commented the code using this model, please help me check.
Thanks you!

**Comment by PaulTR** [comment]
So TFLite might have some trouble with that one op. I'll bring it up with the TFLite team. For now let's remove changing the model and we can add it in later when we have a preferred model that works.

**Comment by PaulTR** [comment]
Did the testing folder names get changed? Getting this error on `pod install` [!] Unable to find a target named `ImageSegmenterUITests` in project `ImageSegmenter.xcodeproj`, did find `ImageSegmenter` and `ImageSegmenterTests`.

File: examples/Image Segmentation/ios/Podfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'ImageSegmenter' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  pod 'TensorFlowLiteTaskVision'
+
+  target 'ImageSegmenterTests' do
```
Comment (PaulTR) [inline examples/Image Segmentation/ios/Podfile]: I deleted these targets and am good to go, so we either need to correct the target names or remove these for building.

**Comment by PaulTR** [comment]
Screenshot 2024-04-22 at 9 54 28 AM

Getting this message in XCode, so just want to verify that the download script is correctly setup.

**Comment by PaulTR** [comment]
Can also update the bottom sheet to consistently show the inference time on the live camera stream.

**Comment by PaulTR** [comment]
Code looks good, just the above comments and this should be merge-ready.

File: examples/Image Segmentation/ios/Podfile (hunk 1 above)
Comment (st-tuanmai) [reply examples/Image Segmentation/ios/Podfile]: I missed it when I deleted the UI tests folder

**Pull request merged**